src/components/Header/DefaultHeader.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+
+import { ClosedLockIcon, UserIcon, LogOutIcon } from '@entur/icons'
+import { useToast } from '@entur/alert'
+
+import { TavlaLogo, Github } from '../../assets/icons'
+import LoginModal from '../LoginModal'
+
+import { useUser } from '../../auth'
+import firebase from 'firebase'
+
+export function DefaultHeader({ theme }: Props): JSX.Element {
+    const [displayLogin, setDisplayLogin] = useState<boolean>(false)
+    const user = useUser()
+    const userLoggedin = !user || !user.isAnonymous
+    const { addToast } = useToast()
+
+    const login = (): void => {
+        event.preventDefault()
+        setDisplayLogin(true)
+    }
+
+    const logout = (): void => {
+        event.preventDefault()
+        addToast({
+            title: 'Logget ut',
+            content: 'Du er nå logget ut av din konto',
+            variant: 'success',
+        })
+        firebase.auth().signOut()
+    }
+
+    const loginModal = !userLoggedin ? (
+        <LoginModal
+            open={displayLogin}
+            onDismiss={(): void => setDisplayLogin(false)}
+        />
+    ) : null
+
+    const userItem = !userLoggedin ? (
+        <div className="header__resources__item" onClick={login}>
+            <p className="header__resources__item__text">Logg inn</p>
```

Comment:
Her kan du sikkert bruke `span` i stedet for `p` ettersom det ikke er et paragraf. Vi pleier å kun bruke `p` kun når  det er tekst som strekker seg over flere linjer :) Gjelder også for linje 47.

Reply:
Du kan også ta å snu på den conditional der og heller bruke: `const userItem = userLoggedIn ? isTrueContent : isFalseContent`

Reply:
Hehe, oppdaget den der nå, aner ikke hva jeg tenkte der. Det skal fikses!